osCam/user/forms.py

```diff
@@ -1,13 +1,7 @@
-from django.db.models.fields import NullBooleanField
 from core.models import *
 from django import forms
-from django.core import validators
 from django.contrib.auth.models import User
 
-from crispy_forms.layout import Field, Layout, Div, HTML, ButtonHolder, Submit
```

Comment:
If we are not going to use crispy forms, can we remove from the requirements.txt

Reply:
for sure. still considering using it, it is affecting the forms in minor ways currently. just not calling any of these specifically.